src/frontend/qt_sdl/Config.cpp

```diff
@@ -347,7 +351,13 @@ ConfigEntry ConfigFile[] =
     {"Camera1_CamDeviceName", 2, &Camera[1].CamDeviceName, (std::string)"", false},
     {"Camera1_XFlip", 1, &Camera[1].XFlip, false, false},
 
-    {"", -1, nullptr, 0, false}
+    {"", -1, nullptr, 0, false},
+
+    {"useCustomDNS", 1, &useCustomDNS, true, true},
```

Comment:
you have to add the new settings after the sentinel value indicating the last entry